examples/statistics/plot_hist.py

```diff
@@ -0,0 +1,101 @@
+"""
+==========
+Histograms
+==========
+
+Demonstrates how to plot histograms with matplotlib.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import colors
+from matplotlib.ticker import PercentFormatter
+
+# Fixing random state for reproducibility
+np.random.seed(19680801)
+
+
+###############################################################################
```

Comment:
OK, turns out these are E265 and E266, but we've got them in our global ignore list.